tests/zfs-tests/tests/functional/quota/quota.kshlib

```diff
@@ -57,7 +57,7 @@ function fill_quota
 	$FILE_WRITE -o create -f $MNTPT/$TESTFILE1 -b $BLOCK_SIZE \
 		 -c $write_size -d 0
 	zret=$?
-        [[ $zret -ne EDQUOT ]] && \
+        [[ $zret -ne $EDQUOT ]] && \
```

Comment:
For me this fixes `quota_001_pos` and `quota_003_pos` on Gentoo.
I haven't tried any of the disabled test cases yet.